server/routes/__testutils/appSetup.ts

```diff
@@ -23,7 +25,13 @@ export const user = {
   authSource: 'NOMIS',
 }
 
-export const signedCookiesProvider = jest.fn()
+const signedCookiesProvider = jest.fn()
+
+export const stubCookie = <T>(state: StateOperations<T>, value: T) =>
+  stubRequestCookie(signedCookiesProvider, state, value)
+
+export const stubCookies = (cookies: ExpectedCookie<unknown>[]) => stubRequestCookies(signedCookiesProvider, cookies)
```

Comment:
couple of new util methods which mean we don't have to refer to cookie name everywhere